lms/djangoapps/courseware/views.py

```diff
@@ -77,8 +80,10 @@
 from shoppingcart.utils import is_shopping_cart_enabled
 from openedx.core.djangoapps.self_paced.models import SelfPacedConfiguration
 from student.models import UserTestGroup, CourseEnrollment
+from student.roles import GlobalStaff
 from student.views import is_course_blocked
 from util.cache import cache, cache_if_anonymous
+from util.course_key_utils import from_string_or_404
```

Comment:
@adampalay  After rebasing it's still exist
See [link](https://github.com/edx/edx-platform/blob/master/common/djangoapps/util/course_key_utils.py#L9)